engines/stark/ui/menu/mainmenu.cpp

```diff
@@ -132,6 +132,12 @@ void MainMenuScreen::helpTextHandler(StaticLocationWidget &widget, const Common:
 	}
 }
 
+void MainMenuScreen::creditsHandler() {
+	if (!isDemo()) {
+		StarkUserInterface->requestFMVPlayback("0e02.bbb");
```

Comment:
There is no 0e02.bbb in the demo game data. So demo has no credits video or with a different name?